operations/security_and_authentication/Restrict_Access_to_NCN_Images_S3_Bucket.md

```diff
@@ -0,0 +1,687 @@
+# Procedure to Restrict Network Access to the ncn-images S3 Bucket
+
+The configuration documented in this procedure is intended to prevent user-facing dedicated nodes (UANs, Compute Nodes) for retrieving NCN image content from Ceph S3 Services, as running on storage nodes. Specifically, the controls enacted via this procedure should:
+
+1. Block HAProxy access to the ncn-images bucket if the client is not an NCN (NMN) or PXE booting from the MTL network. This via a HAProxy ACL on the storage servers.
+2. Enable access logging for HAProxy.
+3. Block Rados GW network access (port 8080) to if the client is not an NCN (NMN) or originating from the HMN network. This via iptables rules on the storage servers.
+
+## Limitations
+
+This is not designed to prevent UAIs (if in use) from retrieving NCN image content. 
+
+If a storage node is rebuilt, this procedure (for the rebuilt node) will need to be applied after the rebuild. The same is true if NCNs are added or removed from the system. 
+
+## Prerequisites and Scope
+
+Procedure should be executed after install or upgrade is otherwise complete, but prior to opening the system for user access. 
+
+Unless otherwise noted, the procedure should be run from ncn-m001 (not PIT).  
+
+The procedure was tested on a CSM 1.2.0 GA System. 
+
```

Comment:
Is everything below this intended to be run sequentially? If so, it might be better to make this an ordered list / set up steps instead of headers.